subcommand/connect-init/command.go

```diff
@@ -63,6 +64,9 @@ func (c *Command) init() {
 	c.flagSet.StringVar(&c.flagConsulServiceNamespace, "consul-service-namespace", "", "Consul destination namespace of the service.")
 	c.flagSet.StringVar(&c.flagServiceAccountName, "service-account-name", "", "Service account name on the pod.")
 	c.flagSet.StringVar(&c.flagServiceName, "service-name", "", "Service name as specified via the pod annotation.")
```

Comment:
nit: should we update the description of this one to say that it's the consul service name? No one is really running `--help` on this command, but maybe just for ourselves?

Reply:
Good idea!